addons/binding/org.openhab.binding.tplinksmarthome/src/main/java/org/openhab/binding/tplinksmarthome/handler/SmartHomeHandler.java

```diff
@@ -90,7 +89,9 @@ public void initialize() {
         connection = createConnection(configuration);
         cache = new ExpiringCache<DeviceState>(TimeUnit.SECONDS.toMillis(configuration.refresh), this::refreshCache);
         updateStatus(ThingStatus.UNKNOWN);
-        startAutomaticRefresh(configuration);
+        if (configuration.refresh > 0) {
```

Comment:
It seems to have a min value of 1 in the XML defs, does this make sense? Or is this test only?

Reply:
Correct it for testing only. I've added an extra comment above the line to explain this.